FirefoxPrivateNetworkVPN/ViewControllers/SettingsDataSource.swift

```diff
@@ -92,7 +92,7 @@ extension SettingsDataSource: UITableViewDelegate {
 
         //swiftlint:disable:next trailing_closure
         headerView?.buttonTappedSubject.subscribe(onNext: { [weak self] _ in
-            self?.settingSelected.onNext(.account)
+            self?.settingSelected.onNext(.account(email: self?.account?.user.email))
```

Comment:
should this be self?.account?.user.email or self?.account?.user?.email

Reply:
`User` in `Account` won't be nil, so I think it's fine.

Reply:
Is it possible self or account here are nil?

Reply:
If self or account is nil, then the whole parameter would directly be nil, and we've skipped this inside when email parameter is nil.  👍